src/core/filter/security.go

```diff
@@ -289,7 +279,11 @@ func (it *idTokenReqCtxModifier) Modify(ctx *beegoctx.Context) bool {
 		log.Warning("User matches token's claims is not onboarded.")
 		return false
 	}
-	groupNames := oidc.GroupsFromToken(claims)
+	settings, err := config.OIDCSetting()
+	if err != nil {
+		log.Errorf("Failed to get OIDC settings, error: %v", err)
```

Comment:
If failed to get OIDC settings, should skip to PopulateGroup

Reply:
That's true, I'll make update.